bot.py

```diff
@@ -153,13 +153,13 @@ async def show(ctx: commands.Context, single: Optional[str]): # noqa: D103
 		)
 
 
-@dev_command()
-async def callMinigame(ctx: commands.Context, natdex: str):
+@bot.command()
+async def callMinigame(ctx: commands.Context, natdex: str = ""):
```

Comment:
Should be `@dev_command()`, right?

Reply:
oops. accidentally overwrote that with a stashed change